### DISCUSSION TITLE
Basic Fundamentals of MOJO

### DISCUSSION THREAD

**Discussion by z-kumar** [modular/modular#439] [General]
Sir/Madam, the problem I'm facing is I'm new to programming and I am not able to find any basic stuff about mojo. For example, i am not able to see the data types used in mojo. conversions of data types, etc.
Please help me with your kind response. 
Thanks a lot.

**Comment by alex-smith-ai** [comment]
Hello, I am also new to mojo and am excited to get started as well. I've been around programming for a few years, but definitely relate.

The two places that seem most helpful to me include:

1. ["Hello, Mojo" notebook](https://docs.modular.com/mojo/notebooks/HelloMojo.html). This is where I plan on starting because it seems like a more helpful introduction to the library than a list of everything available.
2. [The Module Index](https://docs.modular.com/mojo/lib.html) has the "Built-in" and "Standard Library" modules.

If anyone else has other/better suggestions, please add them! Also thank you in advance for the help.

**Comment by mojodojodev** [comment]
Hi @z-kumar I've started working on some guides for beginners, very early stages but there's a comment section down the bottom and would appreciate your feedback if you get a chance to try it out:

https://mojodojo.dev/guides/intro-to-mojo/basic-types.html
